README.md

```diff
@@ -1,17 +1,148 @@
-# Vizro
```

Comment:
Is the higher-level README.md just a duplicate of the vizro-core readme for now?

Reply:
Yes, in the end we decided to keep it like that until a second package arrives